Kitodo/src/main/java/org/kitodo/forms/TemplateBaseForm.java

```diff
@@ -0,0 +1,287 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.forms;
+
+import de.sub.goobi.forms.BasisForm;
+import de.sub.goobi.helper.Helper;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Objects;
+
+import javax.faces.model.SelectItem;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.elasticsearch.search.sort.SortBuilders;
+import org.elasticsearch.search.sort.SortOrder;
+import org.kitodo.data.database.beans.BaseBean;
+import org.kitodo.data.database.beans.Project;
+import org.kitodo.data.database.beans.Task;
+import org.kitodo.data.database.beans.User;
+import org.kitodo.data.database.beans.UserGroup;
+import org.kitodo.data.database.exceptions.DAOException;
+import org.kitodo.data.database.helper.enums.TaskEditType;
+import org.kitodo.data.exceptions.DataException;
+import org.kitodo.services.ServiceManager;
+
+public class TemplateBaseForm extends BasisForm {
+
+    private static final long serialVersionUID = 6566567843176821176L;
+    private static final Logger logger = LogManager.getLogger(TemplateForm.class);
```

Comment:
Wrong class name. It should be `TemplateBaseForm.class` instead of `TemplateForm.class` or?